### ISSUE TITLE
Mobile/tablet support for map dragging

### ISSUE THREAD

**Issue by jdbann** [jdbann/table#16]
Map dragging doesn't work on iOS

**Comment by jdbann** [comment]
I've been looking at this today. Implementing it isn't the hard thing - there's very limited information about creating touch events so I can't properly test this. @cpytel Did you come across any handy tricks when you sussed out the initial dragging approach?

**Comment by cpytel** [comment]
@jdbann no, I didn't. I'm assuming you can manually create an event that has the expected structure?

**Comment by cpytel** [comment]
Also, fwiw, not directly related to this, but I was looking at token movement on iPad yesterday. As of the latest iPadOS, native drag and drop works, so token movement is working. The draggable clone is styled weirdly (with a large green + icon and the token gets squished). 

We can use the `:-webkit-drag` psuedoclass to style the drag clone, but it was weird, after making a few changes everything looked perfect, but then I removed all my customizations so that I could try to identify which ones were actually doing it, and it still looked perfect. Then I put things back and none of them had an effect anymore.

I suspect iPadOS safari is doing some caching of stylesheets or something that was making it difficult to test, and I need to dig into it more.

There is some documentation on this at https://developer.apple.com/library/archive/documentation/AppleApplications/Conceptual/SafariJSProgTopics/DragAndDrop.html

**Comment by jdbann** [comment]
I'm still not happy with the drag and drop code. It feels like the right kind of interaction, but the weird allowances we have to make in Firefox, the hack to get the system tests to work...

I've proposed adding a different delete action in #104. I wonder if this could simplify the approach to:

- Token template is shown in drawer
- Dragging template off the drawer means it is now in the map in whatever location
- Delete the token without touching the drawer

I think we could avoid the drag and drop API completely, just using `MouseEvent`s instead. If we're not returning the token to the drawer then we don't need to worry about hit-boxes or anything like that.
